rasa_core/tracker_store.py

```diff
@@ -49,6 +52,18 @@ def retrieve(self, sender_id):
         # type: (Text) -> Optional[DialogueStateTracker]
         raise NotImplementedError()
 
+    def stream_events(self, tracker):
+        # type: (DialogueStateTracker) -> None
+        old_tracker = self.retrieve(tracker.sender_id)
+        offset = len(old_tracker.events) if old_tracker else 0
+        for evt in tracker.events[offset:]:
+            body = {
+                "sender_id": tracker.sender_id,
+                "name": evt.type_name,
```

Comment:
we are missing the complete event here. I don't think we need `name` but rather the whole `evt`.

Reply:
ready for another review 👍